src/composables/transactionToken.ts

```diff
@@ -60,7 +61,9 @@ export const useTransactionToken = (
         : getTxAmountTotal.value(transaction),
       symbol: isAllowance.value ? AETERNITY_SYMBOL : getTxSymbol.value(transaction),
       isReceived: getTxDirection.value(transaction) === 'received',
-      isAe: isAllowance.value || getTxSymbol.value(transaction) === AETERNITY_SYMBOL,
+      isAe: isAllowance.value
```

Comment:
This conditional has both && and or. I suggest to group them to make other developer's life easier.
Also in case if the conditions should be put into separate lines I suggest to put all of them in separate lines. Eg.:

```
something: (
  condition1
  && condition2
  && condition3
)
```